### PR TITLE
Add Performance and Tracing Feature

### PR DESCRIPTION
**Pull request by rpropst**
Adding the performance/tracing feature into the Express demo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,25 +5,43 @@ Show how Sentry works
 - Import/Integrate
 - Configuration
 - Releases/Commits
+- Performance/Tracing
+
+## Setup
+#### Versions
+this was tested on:
```
Comment (thinkocapo) [inline README.md:12]: awesome thanks for including this section.

File: app.js [hunk 2]
```diff
@@ -5,7 +5,23 @@ var app = express();
 
 // Initalize Sentry (import library and instantiate)
 const Sentry = require('@sentry/node');
-Sentry.init({ dsn: 'https://276b9c69b15b41a3ae98d07206889b24@sentry.io/1366275'});
+const Tracing = require('@sentry/tracing');
+
+Sentry.init({
```
Comment (thinkocapo) [inline app.js:10]: great job with the helpful comments 👍

File: app.js [hunk 3]
```diff
@@ -69,8 +87,11 @@ app.post('/checkout', function (req, res) {
 });
 
 app.get('/capture-message', function (req, res) {
+    // Simulate an API call that takes a random amount
```
Comment (thinkocapo) [inline app.js:90]: Sounds like this is the endpoint you tested @rpropst? Could you post a link in the PR to a /capture-message transaction from your express project you recorded?

Also, we may need to add some manual instrumentation in order for the `/checkout` endpoint to produce transaction data similar to what we get in our sentry-demos/flask app. @ndmanvar should we include that on a next PR or include it to this one? let's take a look at those transactions if there are any so far

Richard, have you tried connecting your React App to this Express app and running through the ToolStore demo, to see what the transactions look like?

Comment (ndmanvar) [reply app.js:90]: let's include it on this one.
@rpropst - we should be able to hit this from a react frontend (/checkout)

Comment (rpropst) [reply app.js:90]: Where could I include the link? Here's a link to the main performance area
https://sentry.io/organizations/testorg-az/performance/summary/?project=5518875&query=&statsPeriod=7d&transaction=GET+%2Fcapture-message&unselectedSeries=p100%28%29

A transaction
https://sentry.io/organizations/testorg-az/performance/r2r-express:540126bb063c4ad387a315b127724750/?project=5518875&query=&statsPeriod=7d&transaction=GET+%2Fcapture-message&unselectedSeries=p100%28%29

I did not try connecting with the React app, and the app currently doesn't call this endpoint. Should we add it in? Update checkout? Looking for guidance here.

Comment (rpropst) [reply app.js:90]: https://sentry.io/organizations/testorg-az/performance/r2r-express:01e4cbca5d08498ba4fea4bb7995d8a4/?project=5518875&query=&statsPeriod=24h&transaction=POST+%2Fcheckout&unselectedSeries=p100%28%29

**Review by ndmanvar — APPROVED** [review]

**Pull request merged**